Feature/vmt

azaaza0319 opened the pull request:
Add vmt, ect. features

Commit:
Update links with distance field

Commit:
Changes suggested by roxygen2

Commit:
Update raw data and Links with fields "speed",v "ffspeed", "capacity"

Commit:
Add functions to calculate VMT, VHT, VHD and VOC

dkyleward commented:
After the new year, let's look at why the R CMD Check is failing.

gregmacfarlane commented:
Looking at the R CMD CHECK, it seems that the documentation hasn't been rebuilt since @azaaza0319 added the new functions/data frame, at least. That's where we can start.

Commit:
Rebuild the documentation with added functions and data frame

Commit:
Simplify code

Only need to include the aggregation function in the ifelse tree; everything else can go at the end. This will make fixing and improving simpler later on.

Commit:
Make fields optional

If I'm calculating VMT, I don't need to supply a capacity field.

Commit:
Fix namespace
